assets/js/displayMemberNames.js

```diff
@@ -0,0 +1,35 @@
+import $ from "jquery";
+
+$(document).ready(() => {
+  $('#names').click(function() {
+    changeDisplayNames();
+  });
+});
+
+function changeDisplayNames() {
+  const teamNamesCurrentlyDisplayed = !document.getElementById("names").checked
```

Comment:
Let's use a more specific ID (obviously DOM will need to be updated too)

```suggestion
  const teamNamesCurrentlyDisplayed = !document.getElementById("name_display_toggle").checked
```